__tests__/util/util.js

```diff
@@ -11,3 +11,7 @@ export function getSquareArray() {
 export function getSquareMatrix() {
     return new Matrix(getSquareArray());
 }
+
+test('This is not a test', () => {
```

Comment:
I think that it's at least a little bit clearer for a new contributor where should be the tests